app/src/config.ts

```diff
@@ -45,6 +45,21 @@ const light = signal({
     fontSize: '0.9em',
     padding: '0.5em 0.75em',
   },
+  link: {
+    textDecoration: 'none',
+    default: {
+      color: '#222',
+    },
+    hover: {
+      color: '#222',
+    },
+    active: {
+      color: '#222',
+    },
```

Comment:
Leave these empty. Then use the body color as the default see [this comment](https://github.com/blackmann/stylebender/pull/32/files#r1252302825) in `Link.tsx` for what I mean.

```suggestion
    default: undefined,
    hoverColor: undefined // hover changes cursor to pointer, so it’s not really critical to have a different color on hover, what do you think?
```


Let’s do away with `:active` for now and set `textDecoration` as `underlined

Reply:
I think you have a point, removed it.